scalalib/src/mill/scalalib/publish/SonatypePublisher.scala

```diff
@@ -135,10 +136,15 @@ class SonatypePublisher(uri: String,
   }
 
   // http://central.sonatype.org/pages/working-with-pgp-signatures.html#signing-a-file
-  private def poorMansSign(file: Path, passphrase: String): Path = {
+  private def poorMansSign(file: Path, maybePassphrase: Option[String]): Path = {
     val fileName = file.toString
     import ammonite.ops.ImplicitWd._
-    %("gpg", "--passphrase", passphrase, "--batch", "--yes", "-a", "-b", fileName)
+    maybePassphrase match {
+      case Some(passphrase) =>
+        %("gpg", "--passphrase", passphrase, "--batch", "--yes", "-a", "-b", fileName)
+      case None =>
+        %("gpg", "--batch", "--yes", "-a", "-b", fileName)
```

Comment:
Is it useful to sign artifacts without `gpgPassphrase` ?

Reply:
I honestly don't know if it's useful, but I'm following Haoyi's comment  : 

https://github.com/lihaoyi/mill/issues/345#issuecomment-390849979

Reply:
Oh, I didn't notice that. All good then